ChromeExtension/js/apikeys.js

```diff
@@ -17,8 +17,10 @@ document.addEventListener('DOMContentLoaded', function () {
 
  
 
-    document.getElementById("apiKey").value = localStorage["apiKey"];
-    document.getElementById("apiSecret").value = localStorage["apiSecret"];
+var apiKeysViewModel = {

```

Comment:
function apiKeyViewModel()
{
	var self = this;
	self.apiKey = ko.observable('');
	self.apiSecret = ko.observable('');
	self.loadKey = function()
	{
		var ak= localStorage["apiKey"];
		if(ak == '')
		{
			self.apiKey(ak);
		}


		var as = localStorage["apiKey"];
		if(as == '')
		{
			self.apiSecret(as);
		}
	}
}

var apiModel = new apiKeyViewModel();

 if (typeof testModel === 'undefined') {
        ko.applyBindings(apiModel );
    } else {
        testModel = apiModel;
    }